src/ParseSchema.ts

```diff
@@ -222,12 +224,15 @@ class ParseSchema {
       throw new Error(`${type} is not a valid type.`);
     }
     if (type === 'Pointer') {
-      return this.addPointer(name, options.targetClass, options);
+      return this.addPointer(name, options.targetClass!, options);
     }
     if (type === 'Relation') {
+      // TODO: Why does options exist here?
       return this.addRelation(name, options.targetClass, options);
```

Comment:
`options` here can be removed as it is unused

Reply:
Thank you for the clarification 👍